### ISSUE TITLE
luckperms placeholder api Expansion doesn't copy full prefix

### ISSUE THREAD

**Issue by AK8221** [LuckPerms/LuckPerms#3419]
### Description

placeholder seems to be ignoring some color codes depending on the worlds/letters before it

### Reproduction Steps

![Capture](https://user-images.githubusercontent.com/109050386/178172989-d7d29ea7-d254-4aab-9bfb-25e19d0aa887.PNG)
![Capture1](https://user-images.githubusercontent.com/109050386/178172990-a46bcd59-cfef-49d0-bc9b-75000ce2faea.PNG)
![Capture2](https://user-images.githubusercontent.com/109050386/178172991-0879fe31-4989-4fac-baea-9a49fcb131ec.PNG)
![Capture3](https://user-images.githubusercontent.com/109050386/178172992-f1cf9340-d450-488e-b3b0-03821c72897c.PNG)
![Capture4](https://user-images.githubusercontent.com/109050386/178172993-7dec4969-5212-4580-9919-5d2e0d898cb7.PNG)
![Capture5](https://user-images.githubusercontent.com/109050386/178172994-faf42302-189d-468c-a865-c570056efcf7.PNG)
![Capture6](https://user-images.githubusercontent.com/109050386/178172996-55fc45ce-fc10-4f4a-b525-6fff756670ce.PNG)
![Capture7](https://user-images.githubusercontent.com/109050386/178172997-be2c1877-79e8-4d4a-b53d-c9c6935def26.PNG)
![Capture8](https://user-images.githubusercontent.com/109050386/178172999-0949f3f7-69d9-4b9e-8573-f26c06f6075a.PNG)
![Capture9](https://user-images.githubusercontent.com/109050386/178173000-a1f18ac8-f967-4f4c-bbf3-fcb6737ed244.PNG)

I believe it is a culprit of the luck perms placeholder Expansion as in deluxe hub all I need to do is set the placer holder and it works, that and when I speak in chat the prefix shows up just fine

### Expected Behaviour

for it to not show the color code in the prefix at all
![Capture](https://user-images.githubusercontent.com/109050386/178173436-845bcc7c-81f1-457f-84e7-4d51ee0cdcb8.PNG)
![Capture1](https://user-images.githubusercontent.com/109050386/178173437-263f15b9-0c75-4b8f-b190-e8f9bddbf02a.PNG)
![Capture2](https://user-images.githubusercontent.com/109050386/178173439-53a6141a-086b-41ca-807c-9d3383c248ad.PNG)

### Server Details

git-paper-387 mc 1.18.2

### LuckPerms Version

LuckPerms-Bukkit-5.4.21

### Extra Details

PlaceholderAPI-2.11.
[Expansion-luckperms.zip](https://github.com/LuckPerms/LuckPerms/files/9080342/Expansion-luckperms.zip)
2

**Comment by ZeromaniacMc** [comment]
This is not a problem on LuckPerm's side. LuckPerms saves prefixes and suffixes as you add them, it will not apply color codes or formatting whatsoever. This is the job of other plugins which display this data, for example chat and tablist ones. 
I had this very same problem when I tried PlaceholderAPI 2.11.2. They did something with color interpretation. Downgrading to 2.11.1 fixed this for me. You should ask PAPI support why this happens if you rather keep the latest version of it and aim to fix this.

**Comment by AK8221** [comment]
> 

yes but placeholder api it self is not able to hook into it witch is why you use [Expansion-luckperms.zip](https://github.com/LuckPerms/LuckPerms/files/9080342/Expansion-luckperms.zip)
witch is handled by luckperms

**Comment by ZeromaniacMc** [comment]
The Luckperms expansion itself is not responsible for interpreting/adding colors. The software behind this is Placeholder API. According to my source, the color interpreting line in Placeholder API was removed. Now, plugins which depended on it doing this for placeholders will no longer work properly. With this I mean chat plugins, tablist ones, and any other which do not do colors on placeholders themselves but until now, depended on PAPI doing that as it did for a long time. It is argued that parsing colors is 'out of scope' for PAPI regardless of that it had been doing it until now. So the issue is with Deluxehub having relied on PAPI's translation of alternate color codes which breaks now since that was removed. 
Again, you can downgrade PAPI or ask the developer of Deluxehub to start translating colors him/herself. 
LuckPerms and the expansion may be the ones getting the raw data, but LuckPerms never touched displaying anything. This is why you need a chat plugin to display prefixes in chat and so forth - it just does not touch this for other plugins to freely use the formats they need.

**Comment by lucko** [comment]
Most likely an issue with the scoreboard plugin.
